### ISSUE TITLE
Migrate from 3.6.5 to 4.0.1

### ISSUE THREAD

**Issue by tsvetiligo** [bizley/yii2-migration#93]
migration/update-all with command --excludeTables throws _UnknownPropertyException_

**Comment by bizley** [comment]
There is no `update-all` in 4.0+. Use `update *`.

**Comment by tsvetiligo** [comment]
```
./yii migration/update * 

Yii 2 Migration Generator Tool v4.0.1
 > No matching tables in database.
 > 1 table excluded by the config.
```

**Comment by bizley** [comment]
Are you by any chance using this in docker environment? Because I've just noticed `*` is not passed properly in that case. You must use `migration/update "*"`.

**Comment by tsvetiligo** [comment]
Yes, I use Docker and `migration/update "*"` works for me.
But I still get error on update for any table:
```
ERROR!
 > Wrong data for addColumn method.
```
Any idea why 3.6.5 works fine and 4.0.1 not?

**Comment by bizley** [comment]
V4 has been rewritten almost in total and is more strict in few places.
But this error is disturbing. Could you paste here the migration files of the table you are trying to update?

**Comment by tsvetiligo** [comment]
m200519_132335_create_table_test.php
```
<?php

use yii\db\Migration;

class m200519_132335_create_table_test extends Migration
{

    public function up()
    {
        $tableOptions = null;
        if ($this->db->driverName === 'mysql') {
            $tableOptions = 'CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE=InnoDB';
        }

        $this->createTable('{{%test}}', [
            'string' => $this->string(),
            ], $tableOptions);

        $this->alterColumn('{{%test}}', 'string', $this->string()->notNull());
    }

    public function down()
    {
        $this->dropTable('{{%test}}');
    }
}
```
```
./yii migrate
Yii Migration Tool (based on Yii v2.0.35)

Total 1 new migration to be applied:
	m200519_132335_create_table_test

Apply the above migration? (yes|no) [no]:yes
*** applying m200519_132335_create_table_test
    > create table {{%test}} ... done (time: 0.018s)
    > alter column string in table {{%test}} to string NOT NULL ... done (time: 0.028s)
*** applied m200519_132335_create_table_test (time: 0.063s)


1 migration was applied.

Migrated up successfully.
```
```
./yii migration/update "*"
Yii 2 Migration Generator Tool v4.0.1
 > 1 table excluded by the config.

 > Comparing current table 'test' with its migrations ...ERROR!
 > Wrong data for addColumn method.
root@46b6ea04d013:/srv# 
```

**Comment by bizley** [comment]
Fixed in [4.0.2](https://github.com/bizley/yii2-migration/releases/tag/4.0.2). Thank you for reporting.
